applications/contracts-tool.md

```diff
@@ -0,0 +1,187 @@
+
+Limitations:
+the project does not compile the contracts by itself, contracts are delivered in binary form.
+
+
+### Ecosystem Fit
+
+The project is useful for ecosystem at contracts development stage to track its performance and regressions on Polkadot.
+It is going to be used also to measure ink! language performance by Parity core team.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Sebastian Miasojed
+- Karol Kokoszka
+
+### Contact
+
+- **Contact Name:** Sebastian Miasojed
+- **Contact Email:** s.miasojed@gmail.com
+- **Website:** 
+
+### Legal Structure
+
+- **Registered Address:** Racula-Janusza Korczaka 15B, 66-004 Zielona Góra, Poland, NIP:5961572852
+- **Registered Legal Entity:** Gloslab Sebastian Miasojed
+
+### Team's experience
+
+We combine development and architecting skills from embedded world, cloud systems and apply them to crypto world.
+Until now the team has shown his proficiency aligning smart-bench with newest libraries required by ink! contracts.
+- https://github.com/paritytech/smart-bench/pull/32
+- https://github.com/paritytech/smart-bench/pull/31
+- https://github.com/paritytech/smart-bench/pull/28
+- https://github.com/paritytech/blockstats/pull/22
+
+### Team Code Repos
+
+- https://github.com/smiasojed
+- https://github.com/karolk91
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/sebastian-miasojed-83b6123
+- https://www.linkedin.com/in/karol-kokoszka-a66959103
+
+
+## Development Status :open_book:
+
+Work has been started, smart bench has been updated with new libraries and is able to build and run on test net with ink! contracts.
+- https://github.com/paritytech/smart-bench/pull/32
+- https://github.com/paritytech/smart-bench/pull/31
+- https://github.com/paritytech/smart-bench/pull/28
+- https://github.com/paritytech/blockstats/pull/22
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):**  0.5
+- **Total Costs:** 15000 EUR
+
+### Milestone 1 Smart-bench updated — Basic functionality
+
+- **Estimated duration:** 26 days
+- **FTE:**  0.5
+- **Costs:** 6500 EUR
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | License | GPLv3 |
+| 2. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can spin up test net and run contracts with transactions. |
+| 3. | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 4. | Updated evm contracts | We will update tool with support for newest Moonbeam parachain. |
```

Comment:
What do you mean by "updated EVM contracts"? And is there anything specific to Moonbeam about any of the deliverables that wouldn't also work for other EVM-compatible parachains, or why are you singling out Moonbeam here?